preprocessor:
  downsampling:
    voxel_grid_filter: false
    leaf_size: 0.05
    random_filter: true
    remaining_points: 5000

  transformation:
    rotation_y: -0.75 # - pi / 4

  normal_estimation:
    use_tree_search_method: true
    number_of_neighbours: 10
    # search_radius: 0.03

  distance_filter:
    distance_threshold: 3.0

  normal_filter:
    # We only interested in points who's normal is mainly in the z direction
    allowed_length_x: 0.5
    allowed_length_y: 0.4
    allowed_length_z: 1.0 # 1.0 is max length of a normal vector

debug: true

